docker-compose.yml

```diff
@@ -10,17 +10,25 @@ services:
     env_file:
       - ./firefly-docker.env
   test:
-    image: ipac/firefly:deps
     build:
       context: ../
       dockerfile: firefly/docker/Dockerfile
       target: deps
     command: bash -c "cd firefly && gradle firefly:test"
-    volumes:
+    volumes: &volInfo
       - ../firefly_test_data:/opt/work/firefly_test_data
       - ../firefly:/opt/work/firefly
       - /opt/work/firefly/.gradle
       - /opt/work/firefly/build
       - /opt/work/firefly/jars/build
       - /opt/work/firefly/node_modules
       - ./build/dist:/opt/work/firefly/build/dist/
```

Comment:
I don't understand why these have to be listed as volumns. Maybe we can talk.

Reply:
Good question.  This is tricky.
The syntax is <host-path>:<container-path>
The first line.. I mount all of ./firefly into the container
`../firefly:/opt/work/firefly`

But, I don't want to do that.  I only want the source files, not any of the files generated from build (dirty).
So, I exclude the files I don't want by forcing an empty mount on those paths where `:` is not given.
i.e `/opt/work/firefly/build`